schema/install_schema.go

```diff
@@ -7,65 +7,65 @@ import (
+	Auto                bool           `json:"auto,omitempty" description:"Set to true when installing without Pairing" yaml:"auto,omitempty"`
+	BindMounts          []string       `json:"bind_mounts,omitempty" yaml:"bind_mounts,omitempty"`
+	Bundles             []BundleSchema `json:"bundles,omitempty" description:"Add bundles in runtime" yaml:"bundles,omitempty"`
+	NoFormat            bool           `json:"no_format,omitempty" yaml:"no_format,omitempty"`
+	Device              string         `json:"device,omitempty" pattern:"^(auto|/|(/[a-zA-Z0-9_-]+)+)$" description:"Device for automated installs" examples:"[\"auto\",\"/dev/sda\"]" yaml:"device,omitempty"`
+	EphemeralMounts     []string       `json:"ephemeral_mounts,omitempty" yaml:"ephemeral_mounts,omitempty"`
+	EncryptedPartitions []string       `json:"encrypted_partitions,omitempty" yaml:"encrypted_partitions,omitempty"`
+	Env                 []interface{}  `json:"env,omitempty" yaml:"env,omitempty"`
+	GrubOptionsSchema   `json:"grub_options,omitempty" yaml:"grub_options,omitempty"`
+	Image               string `json:"image,omitempty" description:"Use a different container image for the installation" yaml:"image,omitempty"`
 	PowerManagement
-	SkipEncryptCopyPlugins bool                `json:"skip_copy_kcrypt_plugin,omitempty"`
-	Partitions             ElementalPartitions `json:"partitions,omitempty" mapstructure:"partitions"`
-	GrubDefEntry           string              `json:"grub-entry-name,omitempty" mapstructure:"grub-entry-name"`
-	ExtraPartitions        []*Partition        `json:"extra-partitions,omitempty" mapstructure:"extra-partitions"`
-	Force                  bool                `json:"force,omitempty" mapstructure:"force"`
-	ExtraDirsRootfs        []string            `json:"extra-dirs-rootfs,omitempty" mapstructure:"extra-dirs-rootfs"`
-	Active                 Image               `json:"system,omitempty" mapstructure:"system"`
-	Recovery               Image               `json:"recovery-system,omitempty" mapstructure:"recovery-system"`
-	Passive                Image               `json:"passive,omitempty" mapstructure:"recovery-system"`
+	SkipEncryptCopyPlugins bool                `json:"skip_copy_kcrypt_plugin,omitempty" yaml:"skip_copy_kcrypt_plugin,omitempty"`
+	Partitions             ElementalPartitions `json:"partitions,omitempty" mapstructure:"partitions" yaml:"partitions,omitempty"`
+	GrubDefEntry           string              `json:"grub-entry-name,omitempty" mapstructure:"grub-entry-name" yaml:"grub-entry-name,omitempty"`
+	ExtraPartitions        []*Partition        `json:"extra-partitions,omitempty" mapstructure:"extra-partitions" yaml:"extra-partitions,omitempty"`
+	Force                  bool                `json:"force,omitempty" mapstructure:"force" yaml:"force,omitempty"`
+	ExtraDirsRootfs        []string            `json:"extra-dirs-rootfs,omitempty" yaml:"extra-dirs-rootfs,omitempty" mapstructure:"extra-dirs-rootfs"`
```

Comment:
can you please always use the same order either json, yaml and mapstructure, or json, mapstructure, yaml, but not mixed?

Reply:
I think json, yaml, mapstructure might be better overall but maybe someone else can also weigh in here